turbo/jsonrpc/bor_snapshot.go

```diff
@@ -193,6 +199,96 @@ func (api *BorImpl) GetCurrentValidators() ([]*valset.Validator, error) {
 	return snap.ValidatorSet.Validators, nil
 }
 
+func (api *BorImpl) SendRawTransactionConditional(ctx context.Context, encodedTx hexutility.Bytes, options types.OptionsAA4337) (common.Hash, error) {
+	txn, err := types.DecodeWrappedTransaction(encodedTx)
+	if err != nil {
+		return common.Hash{}, err
+	}
+
+	txTemp, err := api.db.BeginRo(ctx)
```

Comment:
in erigon we using naming: 
`txn` - ethereum transaction
`tx` - databases transaction

Reply:
Thanks for the information. Updated [here](https://github.com/ledgerwatch/erigon/pull/8229/commits/35915f9ee0152af8b7d4df67864c534ed79c48ca).